CONTRIBUTING.md

```diff
@@ -270,6 +273,31 @@ The Android [versionCode](https://developer.android.com/reference/android/R.styl
 
 ## Troubleshooting
 
+### General Troubleshooting
+
+You can access the Android Debug Bridge (adb) directly in the terminal.
+
+adb commands:
+
+- `adb devices` to see all emulators running.
+- `adb -s {emulator name} logcat '*:E'` to see error and log outputs of the emulator in the terminal.
+
+### Errors to look for in the terminal while using ADB logcat
+
+`NODEJS-MOBILE: terminating with uncaught exception of type std::bad_cast: std::bad_cast` this is related to your NDK version. Make sure that you only have one version of the NDK installed on your device. Refer to [Prerequisites](#preReq)
+
+### <span id="abdRestart">`Blank Screen/Stuck on Splash Screen`</span>
+
+This is most likely due to a severed connection between metro (the react-native bundler) and the emulator. You can manually try to reset it in the terminal. Run the following commands:
+
+- `adb devices` copy the emulator name that you are using
+- `adb -s {emulator name} kill-server`
+- `adb -s {emulator name} reverse tcp:8081 tcp:8081`
```

Comment:
```suggestion
- `adb kill-server`
- `adb devices` copy the emulator name that you are using
- `adb -s {emulator name} reverse tcp:8081 tcp:8081`
```

Kill server is for the whole adb service, not for a specific device. When the adb service has crashed and `kill-server` is needed, `adb devices` does not work.